src/features/island/bumpkin/components/Feed.tsx

```diff
@@ -80,7 +81,10 @@ export const Feed: React.FC<Props> = ({ food, onClose, onFeed }) => {
                 <div className="flex justify-center flex-wrap items-center">
                   <img src={heart} className="me-2 w-6" />
                   <span className="text-xs text-shadow text-center">
-                    {`${selected.experience} EXP`}
+                    {`${getFoodExpBoost(
```

Comment:
can we show the [⚡](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjZ0YSl7bb7AhUzK7kGHSYuAj8QFnoECAoQAQ&url=https%3A%2F%2Fpsfonttk.com%2Fdecoratetext%2Fraio-para-nick-free-fire.html&usg=AOvVaw0gQ-0BkvEYvWnTbNspDdxY) hasBoost icon?
![image](https://user-images.githubusercontent.com/94913189/202614468-34335f23-def5-4501-bbde-949abcb4c914.png)

Reply:
just wanna confirm with @adamhannigan and @spencerdezartsmith if we should continue displaying this icon on every boost. 

At first it was fine because there were few boosts from few sources but now it's kinda hard to maintain and be consistent. Does it really add value as Spencer mentioned here

![image](https://user-images.githubusercontent.com/89294757/202629631-0dfe99b8-9d28-4e73-9ad9-f40f8f7f378d.png)

Should we follow through, some screens might get overcrowded like in Feed where exp and time are boosted ie will we put lightning icon on both rows?

![image](https://user-images.githubusercontent.com/89294757/202629931-02f964e9-bee4-4438-b58f-3adfbe1b34ba.png)